src/interfaces/IEthMultiVault.sol

```diff
@@ -542,6 +566,20 @@ interface IEthMultiVault {
     ///       See `getRedeemAssetsAndFees` for more details on the fees charged
     function redeemAtom(uint256 shares, address receiver, uint256 id) external returns (uint256);
 
+    /// @notice redeem shares from a bonding curve atom vault for assets
```

Comment:
3.10 - New *Curve Functions in EthMultiVault Missing From Interface

We have added the new curve related methods to the interface here.